Langs/0000.lang

```diff
@@ -39,14 +39,17 @@ Not Currently Listed as Compatible=
 Listed as Compatible=
 Cores=
 Threads=
-GPT Detected=
-GPT Not Detected=
+Requirement meet.=
```

Comment:
Shouldnt it be "requirements met" in this situation?

Reply:
No.

Requirement = Singular (one)
Requirements = Plural (two+)

I think we check many requiremen*ts*, but in this check only one requiremen*t* must be fulfilled.

Reply:
Met, not meet.

Reply:
Ooops.